exercises/robot-simulator/src/test/java/RobotTest.java

```diff
@@ -6,19 +6,21 @@
 public class RobotTest {
 
     @Test
-    public void testRobotIsCreatedWithCorrectInitialPosition() {
+    public void testRobotIsCreatedWithCorrectInitialPositionAndOrientation() {
         final GridPosition initialGridPosition = new GridPosition(0, 0);
         final Robot robot = new Robot(initialGridPosition, Orientation.NORTH);
 
+        assertEquals(robot.getOrientation(), initialOrientation);
```

Comment:
You need to specify `initialOrientation` in the same way as intitialGridPosition :)